tests/wicked/advanced_ref.pm

```diff
@@ -43,12 +43,16 @@ sub run {
 
     record_info('Test 3', 'Create a SIT interface with IP commands');
     $self->create_tunnel_with_commands("sit1", "127");
-    # Lock until parent creates mutex 'test_1_ready'
+    # Lock until parent creates mutex 'test_3_ready'
     mutex_lock('test_3_ready');
```

Comment:
You can use `mutex_wait`, which is wrapper for lock & unlock

Reply:
Already tested with mutex_wait(). Thanks!